src/BusinessLayer/Services/TicketsService.cs

```diff
@@ -102,22 +102,32 @@ public TicketsService([NotNull] ITicketsRepository ticketsRepository)
             return await GetTicketById(ticketId) ?? throw new InvalidOperationException();
         }
 
-        public async void CreateTicket(TicketBlModelRequest ticket)
+        public async Task<TicketBlModelResponse> CreateTicket(TicketBlModelRequest ticket)
         {
             TicketDalDtoModelRequest ticketRequest = new TicketDalDtoModelRequest
                 (
                     ticket.UserId,
                     ticket.PriceId
                 );
 
-            int id = await _ticketsRepository.CreateTicket(ticketRequest);
+            Tuple<DataAccessLayer.StoredProcedureExecutionResult, int> resultTuple
+                = await _ticketsRepository.CreateTicket(ticketRequest);
 
-            if (ticket.Services != null)
+            if (resultTuple.Item1 == DataAccessLayer.StoredProcedureExecutionResult.Ok)
```

Comment:
If you use tuples please use ValueTuples. Item1 is not something meaningful. Also it is usually bad practise to use Tuples as contracts.